bedrock/externalpages/templates/externalpages/pocket/save-to-pocket.html

```diff
@@ -0,0 +1,92 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "externalpages/pocket/base.html" %}
+
+{% from "macros-protocol.html" import split with context %}
+
+{% block pocket_css %}
+  {{ css_bundle('pocket-features') }}
+{% endblock  %}
+
+{% block page_title %}Save to Pocket{% endblock %}
+
+{% block content %}
+
+{% include 'externalpages/pocket/includes/nav.html' %}
+
+<section>
```

Comment:
**Nit** (non-blocking): The padding in this section is off when compared to prod